Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "TreeParzen"
 uuid = "eb66a70c-a255-11e9-03ea-7ba6b2f22006"
 authors = ["iqml <https://github.com/iqml>"]
-version = "0.1.44"
+version = "0.1"
```

Comment:
Should be 0.1.0 I believe, even if 0.1 works correctly, better to be explicit

Reply:
a41e329 resolves